Updated German catkeys file

humdingerb opened the pull request.

Commit:
Updated German catkeys file

puckipedia commented on CatKeysEditor/locales/de.catkeys:
```diff
@@ -10,4 +11,7 @@ Filter:	TranslationView		Filter:
+The file failed to load. Are you sure it's a catkey file?	CatKeyApp		Die Datei konnte nicht geladen werden. Handelt es sich tatsächlich um eine catkeys Datei?
+Failed to load file	CatKeyApp		Fehler beim Laden der Datei
 Failed to suggest the translation.\nDo you have the right permissions?	TranslationView		Übersetzungsvorschlag fehlgeschlagen.\nVielleicht fehlen die nötigen Berechtigungen?
+ngen?
```
Woops! That shouldn't be there!

Commit:
Removed garbage at the end of de. Changed nl to "Dutch".

Commit:
Changed string "Translated" -> "Translation"

Let's stick to nouns as headings.
Generated catkeys anew. Hope "Vertaling" is the correct noun for NL.
